interfaces/core.go

```diff
@@ -78,23 +78,23 @@ type Interface interface {
 	// SanitizeSlot checks if a slot is correct, altering if necessary.
 	SanitizeSlot(slot *Slot) error
 
-	// PlugSecuritySnippet returns the configuration snippet needed by the
-	// given security system to allow a snap to offer a plug of this interface.
+	// SlotSecuritySnippet returns the configuration snippet needed by the
+	// given security system to allow a snap to offer a slot of this interface.
 	//
-	// An empty snippet is returned when the plug doesn't require anything
+	// An empty snippet is returned when the slot doesn't require anything
 	// from the security system to work, in addition to the default
-	// configuration.  ErrUnknownSecurity is returned when the plug cannot
+	// configuration.  ErrUnknownSecurity is returned when the slot cannot
 	// deal with the requested security system.
-	PlugSecuritySnippet(plug *Plug, slot *Slot, securitySystem SecuritySystem) ([]byte, error)
+	SlotSecuritySnippet(plug *Plug, slot *Slot, securitySystem SecuritySystem) ([]byte, error)
 
-	// SlotSecuritySnippet returns the configuration snippet needed by the
-	// given security system to allow a snap to use a plug of this interface.
+	// PlugSecuritySnippet returns the configuration snippet needed by the
+	// given security system to allow a snap to use a slot of this interface.
 	//
 	// An empty snippet is returned when the plug doesn't require anything
 	// from the security system to work, in addition to the default
 	// configuration.  ErrUnknownSecurity is returned when the plug cannot
 	// deal with the requested security system.
```

Comment:
This is confusingly correct IMHO. I re-wrote that paragraph to be much less confusing in the upcoming static security snippets branch.

Here both sides talk about a slot because you "consume a slot" or "provide a slot" and this is exactly what those bits were initially meant to do. I'd ignore it and review the wording on the static/dynamic branch